resources/hu/cemu.po

```diff
@@ -1317,15 +1364,15 @@ msgstr "A Cemu VRAM használata MB-ban"
 
 #: GeneralSettings2.cpp
 msgid "This option requires Win8.1+"
-msgstr "Ehhez a funkcióhoz Win8.1 vagy későbbi rendszer kell"
+msgstr "Ehhez a funkcióhoz Win8.1 vagy újabb rendszer kell"
 
 #: GeneralSettings2.cpp
 msgid "Debug"
-msgstr "Debuggolás"
+msgstr "Hibakeresés"
```

Comment:
I've noticed that in some cases you translated "Debug" (and words derived from it) into "Hibakeresés", and in other places into "Hibaelhárítás".
Is it intentional or did you make your mind at some point and forgot to change what you did earlier?

Reply:
Yeah, that was my mistake. It should be „hibakeresés” everywhere. I'll fix that now.